lib/unbrick.js

```diff
@@ -0,0 +1,27 @@
+var update = require('./update-fetch');
+var logs = require('./logs');
+var flash = require('./flash');
+
+module.exports = function unbrick(opts, tessel) {
```

Comment:
For the sake of consistency with other commands, I recommend making the `unbrick` command a method on the `Tessel` prototype. We have been doing this for all commands that actually manipulate the Tessel. See [the version command](https://github.com/tessel/t2-cli/blob/master/lib/tessel/version.js) as an example. If you do this, please move this file to `lib/tessel/unbrick.js`.